experiments/snippets/kanbanBoards/TaskManagementKanban.xml

```diff
@@ -0,0 +1,1011 @@
+<skuidpage showsidebar="false" showheader="false" tabtooverride="Lead" personalizationmode="server" theme="PlainRobotoTheme">
```

Comment:
Ah, didn't notice this the first time, but this page uses a custom theme. Let's remove it.
```suggestion
<skuidpage showsidebar="false" showheader="false" tabtooverride="Lead" personalizationmode="server">
```

Reply:
Removed in next commit.